### PR TITLE
Merge dev-pt into main

### PR DESCRIPTION
**Pull request by edsoncelio**
Merge Portuguese development branch in main.

### CODE REVIEW

**Comment by edsoncelio** [comment]
This will continue in draft until we fix all the conflicts :)

**Comment by jihoon-seo** [comment]
Thank you all for your devotion and hard work! :)

**Comment by seokho-son** [comment]
@edsoncelio Unlocked ! :)

**Comment by seokho-son** [comment]
Check list for maintainers for a PR to make localization live.
 - [x] validate all files which are not in the localization directory (content/xx/*)
 - [x] check i18n/xx.toml exist
 - [x] check l10n configuration in config.toml
 - [x] acknowledge to initial localization contributors list
 - [x] check Preview works

**Comment by seokho-son** [comment]
@edsoncelio 

https://github.com/cncf/glossary/blob/40729b8697c1bd057449c7871e59b2dc1fc50731/i18n/en.toml#L63

> # Feedback section
> [feedback_title]
> other = "Feedback"
> [feedback_question]
> other = "Was this page helpful?"
> [feedback_answer_yes]
> other = "Yes"
> [feedback_answer_no]
> other = "No"

pt-br.toml can be updated for Feedback section. #470

I think you can open another PR to dev-pt. :)

**Comment by seokho-son** [comment]
@edsoncelio 

https://github.com/cncf/glossary/blob/40729b8697c1bd057449c7871e59b2dc1fc50731/i18n/en.toml#L63

> # Feedback section
> [feedback_title]
> other = "Feedback"
> [feedback_question]
> other = "Was this page helpful?"
> [feedback_answer_yes]
> other = "Yes"
> [feedback_answer_no]
> other = "No"

pt-br.toml can be updated for Feedback section. #470

I think you can open another PR to dev-pt. :)

**Comment by edsoncelio** [comment]
> @edsoncelio
> 
> https://github.com/cncf/glossary/blob/40729b8697c1bd057449c7871e59b2dc1fc50731/i18n/en.toml#L63
> 
> > # Feedback section
> > [feedback_title]
> > other = "Feedback"
> > [feedback_question]
> > other = "Was this page helpful?"
> > [feedback_answer_yes]
> > other = "Yes"
> > [feedback_answer_no]
> > other = "No"
> 
> pt-br.toml can be updated for Feedback section. #470
> 
> I think you can open another PR to dev-pt. :)

Done! I've created another PR to `dev-pt` and everything looks good, to check #710

**Comment by edsoncelio** [comment]
Preview:
image

**Comment by seokho-son** [comment]
I am waiting for #710 :)

**Review by seokho-son — APPROVED** [review]
LGGGGGGGTM :) 

Thank you ! Portuguese l10n team !!

File: content/pt-br/style-guide/_index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Cada definição do glossário é escrita em um arquivo markdown, baseada no seguinte modelo:
+
+```md
+---
+title: 
+status: 
+category: 
+---
+
+## What it is
```
Comment (iamNoah1) [inline content/pt-br/style-guide/_index.md:40]: I guess this is not correct?!

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
I left a comment.

File: content/pt-br/style-guide/_index.md (hunk 1 above)
Comment (seokho-son) [reply content/pt-br/style-guide/_index.md:40]: @iamNoah1 I think it is correct :) 

`code` applied from line 40 ~ line 51 

PTAL: https://deploy-preview-675--cncfglossary.netlify.app/pt-br/style-guide/#modelo-para-as-defini%C3%A7%C3%B5es

Comment (edsoncelio) [reply content/pt-br/style-guide/_index.md:40]: Hi @iamNoah1! As Seokho said, this is a code block to show the original structure of a definition file, On the [Definition topic](https://deploy-preview-675--cncfglossary.netlify.app/pt-br/style-guide/#subcabe%C3%A7alhos) I've translated the subheading to Portuguese :)

**Review by JasonMorgan — APPROVED** [review]

File: content/pt-br/style-guide/_index.md (hunk 1 above)
Comment (edsoncelio) [reply content/pt-br/style-guide/_index.md:40]: No worries, thanks for your detailed review @iamNoah1!

**Review by jessicalins — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**